src/Components/SingleArticle.jsx

```diff
@@ -8,27 +8,37 @@ import Card from '@mui/material/Card';
 import CardContent from '@mui/material/CardContent';
 import CardMedia from '@mui/material/CardMedia';
 import Typography from '@mui/material/Typography';
+import VoteButton from './VoteButton'
 
 export default function SingleArticle(){
 
   const { articleId } = useParams();
   const [article, setArticle] = useState(null);
   const [loading, setLoading] = useState(true);
   const [comments, setComments] = useState([]);
+  const [totalVotes, setTotalVotes] = useState(0);
 
   useEffect(() => {
     Promise.all([getArticleById(articleId), getCommentsById(articleId)])
     .then(([articleData, commentsData]) => {
       setArticle(articleData);
       setComments(commentsData);
       setLoading(false);
```

Comment:
have a separate loading state for the articles and the comments once you've separated them

Reply:
Ok makes sense. I have updated them into two separate promises